srcpkgs/sysbench/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'sysbench'
+pkgname=sysbench
+version=1.0.18
+revision=1
+archs="x86_64"
+build_style=configure
+configure_args="--prefix=/usr --with-pgsql"
+hostmakedepends="make automake libtool pkg-config"
```

Comment:
make is already installed in the build chroot, it does not need to be explicitely listed.